logprep/connector/opensearch/output.py

```diff
@@ -87,16 +89,21 @@ class Config(ElasticsearchOutput.Config):
             default=4, validator=[validators.instance_of(int), validators.gt(1)]
         )
         """Number of threads to use for bulk requests."""
-
         queue_size: int = field(
             default=4, validator=[validators.instance_of(int), validators.gt(1)]
         )
         """Number of queue size to use for bulk requests."""
-
         chunk_size: int = field(
             default=500, validator=[validators.instance_of(int), validators.gt(1)]
         )
         """Chunk size to use for bulk requests."""
+        bulk_retries: int = field(
+            default=5, validator=[validators.instance_of(int), validators.gt(1)]
+        )
+        """Number of retries to use for bulk requests, if OpenSearch returns a 429
+        rejected_execution_exception. Between each retry there will be a small delay, starting
+        with one second. With each consecutive retry 500 to 1000 ms will be added to the delay,
+        chosen randomly. :code:`bulk_retries` is optional and has a default value of 5."""
```

Comment:
you have the `max_retries` parameter on this Component (have a look in elasticsearch_output). Why not use this parameter?